tests/app/templating/test_template_renderer.py

```diff
@@ -91,3 +91,31 @@ def test_render_nested_templatable_property(self):
         }
 
         self.assertEqual(rendered, expected)
+
+    def test_given_none_safe_content_should_return_none(self):
+        content = None
+
+        safe_content = TemplateRenderer.safe_content(content)
+
+        self.assertEqual(safe_content, None)
+
+    def test_given_empty_string_safe_content_should_return_empty_string(self):
+        content = ''
+
+        safe_content = TemplateRenderer.safe_content(content)
+
+        self.assertEqual(safe_content, '')
+
+    def test_should_replace_jinja_template_variable(self):
+        content = 'Your answer is {{ answers.your_answer}}'
```

Comment:
Worth checking some other string combinations that are valid e.g. pipe symbol is used often as well as brackets e.g. {{something|length}} or {{something(else, a, b)}} just to cover a few scenarios from our current schemas (maybe worth pull some of these out of UKIS, Census etc.